protocol/avdl/keybase1/kbfs.avdl

```diff
@@ -25,4 +25,15 @@ protocol kbfs {
     given requestID.
         */
   void FSEditList(array<FSNotification> edits, int requestID);
+
+  /**
+    kbfs calls this as a response to receiving an FSSyncStatusRequest with a
+    given requestID.
+        */
+  void FSSyncStatus(FSSyncStatus status, int requestID);
```

Comment:
Was just keeping the file internally consistent, but I'll fix this new one if you prefer it.